web/pgadmin/browser/templates/browser/js/utils.js

```diff
@@ -20,7 +20,7 @@
   {% endif %}label: "{{ item.label }}", applies: ["{{ key.lower() }}"],
   priority: {{ item.priority }},
   enable: "{{ item.enable }}",
-  {% if item.checked is defined %}checked: {% if item.checked %}true{% else %}false{% endif %},
+  {% if item.checked is defined %}checked: {% if item.name == 'mnu_lock_'+current_ui_lock %}true{% else %}false{% endif %},
```

Comment:
There should be one more condition for `{% if item.checked %}`

Reply:
iten.checked will either be true or false and  was earlier used  to set checked property to true or false, I think adding item.checked will not have any effect. please let me know if I am missing any point here.